package.json

```diff
@@ -220,15 +220,15 @@
     "test": "run-s -s test-client test-server",
     "pretest-client": "npm run -s pretest",
     "test-client": "jest -c=test/client/jest.config.json",
-    "test-client:ci": "cross-env-shell TEST_REPORT_FILENAME=./test-results-client.xml jest -c=test/client/jest.config.ci.js -w=2",
+    "test-client:ci": "node bin/ci-test-runner.js client",
```

Comment:
it stinks when we can't use regular CLIs and need to create custom runners.
but its a bug, and this fixes it.

so it goes